pkg/util/paramtable/service_param.go

```diff
@@ -263,6 +265,72 @@ We recommend using version 1.2 and above.`,
 	p.EtcdTLSMinVersion.Init(base.mgr)
 }
 
+// /////////////////////////////////////////////////////////////////////////////
+// --- tikv ---
+type TiKVConfig struct {
+	Endpoints    ParamItem          `refreshable:"false"`
+	RootPath     ParamItem          `refreshable:"false"`
+	MetaSubPath  ParamItem          `refreshable:"false"`
+	KvSubPath    ParamItem          `refreshable:"false"`
+	MetaRootPath CompositeParamItem `refreshable:"false"`
+	KvRootPath   CompositeParamItem `refreshable:"false"`
+}
+
+func (p *TiKVConfig) Init(base *BaseTable) {
+	p.Endpoints = ParamItem{
+		Key:          "tikv.endpoints",
+		Version:      "7.2.0",
```

Comment:
`Version` means this `ParamItem` effective since which Milvus release. Maybe value should be "2.3.0" here?

Reply:
Good to know that. Done.